frontend/src/components/Search/SearchForm/SearchForm.tsx

```diff
@@ -494,21 +493,21 @@ export const Search: React.FC<{
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [query])
 
+	const { items, open } = comboboxStore.getState()
+
 	useEffect(() => {
-		// Logic for selecting a default item from the results. We don't want to
-		// select a value by default, but if there is a query, an item isn't
-		// currently selected, and there are items, select the first item.
-		const { activeId, items } = comboboxStore.getState()
-		// Give preference to the "Show all results for..." item if it exists.
-		const firstItem = items.find((i) => i.value === undefined) ?? items[0]
-		const noActiveId = !activeId || !items.find((i) => i.id === activeId)
-
-		if (activePart.text.trim() !== '' && noActiveId && firstItem) {
+		if (!open) {
+			return
+		}
```

Comment:
some flickering when loading for sessions / error https://www.loom.com/share/f129e09f49a74fdf9a8058c071bc443c

Reply:
Thanks for catching! I updated the initial cursor index to be the length of the query - what was happening was we were defaulting to 0, which caused us to fetch the values of the first key (happened for sessions and errors since we have a default search)